src/item.cpp

```diff
@@ -1930,9 +1930,14 @@ std::string item::info( std::vector<iteminfo> &info, const iteminfo_query *parts
                 if( g->u.get_skill_level_object( book.skill ).can_train() &&
                     parts->test( iteminfo_parts::BOOK_SKILLRANGE_MAX ) ) {
                     auto fmt = string_format(
-                                   _( "Can bring your <info>%s skill to</info> <num>" ),
+                                   _( "Can bring your <info>%s skill to</info> <num>." ),
                                    book.skill.obj().name() );
                     info.push_back( iteminfo( "BOOK", "", fmt, iteminfo::no_flags, book.level ) );
+                    fmt = string_format(
+                              _( "Your current <stat>%s skill</stat> is <num>." ),
+                              book.skill.obj().name() );
+                    info.push_back( iteminfo( "BOOK", "", fmt, iteminfo::no_flags,
+                                              g->u.get_skill_level( book.skill.obj().ident() ) ) );
```

Comment:
Same here.

Reply:
Which thing "same here" ?

Reply:
The comments were oddly reordered after submitting the review (I guess the order of creation mattered). I meant replacing ` g->u.get_skill_level( book.skill.obj().ident() ) )` with a temp where it's stashed.

Reply:
Is it really necessary to stash it if it's only used once, though?

Reply:
No, not really, but not stashing it is tiny code duplication with all its drawbacks (scaled accordingly, but still).

Reply:
I don't understand how not stashing it is any kind of code duplication for something only used once...

Reply:
It would be _reused_ once, but used twice. Note the condition above (`... g->u.get_skill_level_object( book.skill ).can_train() && ...`).

Reply:
The problem was I wasn't perceiving that it was being used in more than one place because my brain tends to perceive things like that as discrete things.